python/extensions/pip.bzl

```diff
@@ -14,6 +14,6 @@
 
 "pip module extension for use with bzlmod"
 
-load("//python/private/bzlmod:pip.bzl", _pip = "pip")
+load("//python/private/pypi:pip.bzl", _pip = "pip")
```

Comment:
nit: maybe we should rename the `pypi:bzlmod.bzl` to `pypi:extension.bzl` and use that instead of `pypi:pip.bzl`? It does seem like we could benefit from this cleanup.

Reply:
SGTM. Will rename.